server/historian/packages/historian-base/src/utils.ts

```diff
@@ -87,25 +87,33 @@ export function parseToken(tenantId: string, authorization: string): string {
  * Pass into `.catch()` block of a RestWrapper call to output a more standardized network error.
  * @param url request url to be output in error log
  * @param method request method (e.g. "GET", "POST") to be output in error log
- * @param networkErrorOverride NetworkError to throw, regardless of error received from request
+ * @param lumberProperties Collection of Lumberjack telemetry properties associated with the request
  */
 export function getRequestErrorTranslator(
     url: string,
     method: string,
     lumberProperties?: Map<string, any> | Record<string, any>): (error: any) => never {
     const standardLogErrorMessage = `[${method}] Request to [${url}] failed`;
     const requestErrorTranslator = (error: any): never => {
-        // BasicRestWrapper only throws `AxiosError.response.status` when available.
-        // Only bubble the error code, but log additional details for debugging purposes
-        if (typeof error === "number" || !Number.isNaN(Number.parseInt(error, 10)))  {
+        if (typeof error === "object" && error instanceof Error && error.name === "NetworkError") {
+            // BasicRestWrapper throws NetworkErrors that decribe the error details associated with the network call.
+            // Here, we log information associated with the error for debugging purposes, and re-throw.
+            const networkError = error as NetworkError;
+            // eslint-disable-next-line max-len
+            winston.error(`${standardLogErrorMessage}: [statusCode: ${networkError.code}], [message: ${networkError.message ?? "undefined"}]`);
```

Comment:
do we also need to log networkError.details here along with or instead of networkError.details message?

Reply:
Good point. I updated that to be `networkError.details` instead - it includes all details, including message :)